src/components/TvDetails/index.tsx

```diff
@@ -393,9 +393,19 @@ const TvDetails: React.FC<TvDetailsProps> = ({ tv }) => {
               </span>
             )}
           </h1>
-          <span className="mt-1 text-xs lg:text-base lg:mt-0">
+          <div className="mt-1 text-xs lg:text-base lg:mt-0">
             {data.genres.map((g) => g.name).join(', ')}
-          </span>
+            {data.contentRatings.results.length > 0 && (
```

Comment:
Create a `seriesAttributes` array similar to `movieAttributes` in `MovieDetails`

Reply:
Got it ;)